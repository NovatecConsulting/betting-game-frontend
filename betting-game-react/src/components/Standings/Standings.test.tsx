import React from "react";
import { shallow } from "enzyme";
import { Standings } from ".";

describe(Standings.name + " component", () => {
    it("should render and match snapshot with correct content text.", () => {
        const wrapper = shallow(<Standings />);

        expect(wrapper.find("div")).toIncludeText("Standings Component");
        expect(wrapper).toMatchSnapshot();
    });
});
