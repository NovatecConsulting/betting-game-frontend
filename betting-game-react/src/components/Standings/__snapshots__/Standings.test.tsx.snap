// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Standings component should render and match snapshot with correct content text. 1`] = `
<div
  className="min-h-screen min-w-screen bg-white text-center text-6xl text-black grid items-center tracking-wider"
>
  Standings Component
</div>
`;
